tests/compressibility_lateral_pipe_advect_projected_density/statistics

```diff
@@ -31,4 +31,4 @@
 3 1.235979533507e+06 3.949094213442e+05 8 126 72 72 4 3 4294967292          0  7  7                                                                                   "" 1.31708858e-01 1.31710491e-01 1.33783333e+03 1.51464294e+03 1.62471164e+03 5.05170320e-02 3.30564248e+03 1.00000000e+21 3.30564248e+14 -4.34422994e+07 4.36928706e+07 0.00000000e+00 0.00000000e+00 
 4 1.615600013561e+06 3.796204800533e+05 8 126 72 72 4 3 4294967292          0  7  7                                                                                   "" 1.36804753e-01 1.36805705e-01 1.36842574e+03 1.52868107e+03 1.63230525e+03 4.57872383e-02 3.30471393e+03 1.00000000e+21 3.30471393e+14 -4.51163083e+07 4.53554116e+07 0.00000000e+00 0.00000000e+00 
 5 1.981081846813e+06 3.654818332529e+05 8 126 72 72 4 3 4294967292          0  7  7                                                                                   "" 1.41897120e-01 1.41897710e-01 1.39478921e+03 1.54164751e+03 1.63961605e+03 4.19595128e-02 3.30385658e+03 1.00000000e+21 3.30385658e+14 -4.67888886e+07 4.70187969e+07 0.00000000e+00 0.00000000e+00 
-6 2.000000000000e+06 1.891815318658e+04 8 126 72 72 4 3 4294967292          0  7  7                                                                                   "" 1.42165816e-01 1.42166402e-01 1.39607382e+03 1.54230426e+03 1.63999448e+03 4.17801274e-02 3.30381316e+03 1.00000000e+21 3.30381316e+14 -4.68771335e+07 4.71066214e+07 0.00000000e+00 0.00000000e+00 
+6 2.000000000000e+06 1.891815318658e+04 8 126 72 72 4 3 4294967292 4294967295  5  5                                                                                   "" 1.42165816e-01 1.42166402e-01 1.39607382e+03 1.54230426e+03 1.63999448e+03 4.17801274e-02 3.30381316e+03 1.00000000e+21 3.30381316e+14 -4.68771335e+07 4.71066214e+07 0.00000000e+00 0.00000000e+00 
```

Comment:
Not related to this PR, but is there a reason the number of iterations for the composition and stokes solves are at or near the max 32 bit unsigned int values?